drivers/ukconsole/ns16550/Config.uk

```diff
@@ -0,0 +1,141 @@
+menuconfig LIBUKCONSOLE_NS16550
+	bool "NS16550 / 8250"
+	depends on (ARCH_ARM_64 || ARCH_X86_64)
+	select LIBUKOFW if ARCH_ARM_64
+    select LIBUKCONSOLE
+
+if LIBUKCONSOLE_NS16550
+
+if ARCH_ARM_64
+
+config LIBUKCONSOLE_NS16550_EARLY_CONSOLE
+	bool "Early console"
+	select LIBUKBOOT
+	help
+		Uses the fdt's `/chosen/stdout-path` to derive the
+		default console. Override by setting `ns16550.base`
+		at the cmdline, if LIBUKLIBPARAM is enabled.
+endif
+
+if ARCH_X86_64
+
+config LIBUKCONSOLE_COM_EARLY_CONSOLE
```

Comment:
Regarding the last update that moves the `com` into `ns16550`. I think you need to update the config directives to match the rest of the driver, i.e. `CONFIG_UKCONSOLE_NS16550_...`. Indentation in `Config.uk` must use tabs.

Also remember to update the `Config.uk` of KVM, as currently no console drivers is selected for `x86_64`.